internal/kubernetes/operator/features/crds.go

```diff
@@ -46,25 +49,40 @@ var (
 	ErrDocumentHasNoSchema       = errors.New("document contains no Schema")
 	ErrDocumentHasNoSpec         = errors.New("document contains no Spec")
 
-	versionsToResourcesMap = map[string][]string{
+	versionsToResourcesMap = map[string][]resource{
 		"2.0.0": {
-			ResourceAtlasDatabaseUser,
-			ResourceAtlasProject,
-			ResourceAtlasDeployment,
-			ResourceAtlasBackupSchedule,
-			ResourceAtlasBackupPolicy,
-			ResourceAtlasTeam,
-			ResourceAtlasDataFederation,
-			ResourceAtlasFederatedAuth,
+			resource{ResourceAtlasDatabaseUser, NopPatcher()},
+			resource{ResourceAtlasProject, NopPatcher()},
+			resource{ResourceAtlasDeployment, NopPatcher()},
+			resource{ResourceAtlasBackupSchedule, NopPatcher()},
+			resource{ResourceAtlasBackupPolicy, UnkownBackupPolicyFrequencyTypesPruner()},
```

Comment:
In the future we might need to pipeline patchers, if more than one patch applies to a CRD. Not a big deal, I guess we can create a PatcherAggregator that takes several patchers are arguments and applies them in order.

Reply:
yes, it would look like this:

```
func MultiPatcher(patcher ...Patcher) Patcher {
	return PatcherFunc(func(crdSpec *v1.JSONSchemaProps, obj runtime.Object) error {
		for _, p := range patcher {
			if err := p.Patch(crdSpec, obj); err != nil {
				return err
			}
		}
		return nil
	})
}
```

however, linter would complain as this function is currently unused